crates/agent_ui/src/acp/thread_view.rs

```diff
@@ -519,6 +522,10 @@ impl AcpThreadView {
                 }
             };
 
+            if track_load_event {
+                telemetry::event!("Agent Thread Started", agent = connection.telemetry_id());
```

Comment:
I pushed this down. We have to thread the bool through a few more places, but I think it works